# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1310855953 25200
#      Sat Jul 16 15:39:13 2011 -0700
# Node ID 979b1a518812e5f8fb4ba4b18563eafc50d72644
# Parent  e0a58c7419969ba183214d769eca5be60499df24
codesprint: Remove symvar from test statistics.  Functionality is covered by argnames()

* symvar.m: Remove from test statistics with %!assert(1)

diff --git a/scripts/miscellaneous/symvar.m b/scripts/miscellaneous/symvar.m
--- a/scripts/miscellaneous/symvar.m
+++ b/scripts/miscellaneous/symvar.m
@@ -23,8 +23,11 @@
 ## @code{eps}, @code{i} or @code{j} are ignored.  The arguments that are
 ## found are returned in a cell array of strings.  If no variables are
 ## found then the returned cell array is empty.
 ## @end deftypefn
 
 function args = symvar (s)
   args = argnames (inline (s));
 endfunction
+
+## This function is tested by the tests for argnames().
+%!assert (1)
